File: MedLabO.Models/Novost.cs
namespace MedLabO.Models
{
    public class Novost
    {
        public string NovostID { get; set; }
        public string Naslov { get; set; }
        public string Sadrzaj { get; set; }
        public DateTime DTKreiranja { get; set; }
        public DateTime? DTZadnjeModifikacije { get; set; }
        public byte[] Slika { get; set; }
        public string? AdministratorID { get; set; }
        public virtual Administrator? Administrator { get; set; }
    }
}

File: MedLabO UI/medlabo_mobile/lib/models/novost/novost.dart
import 'package:json_annotation/json_annotation.dart';

part 'novost.g.dart';

@JsonSerializable()
class Novost {
  String? novostID;
  String? naslov;
  String? sadrzaj;
  String? dtKreiranja;
  String? dtZadnjeModifikacije;
  String? slika;
  String? administratorID;

  Novost(
      {this.novostID,
      this.naslov,
      this.sadrzaj,
      this.dtKreiranja,
      this.dtZadnjeModifikacije,
      this.slika,
      this.administratorID});

  factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);

  Map<String, dynamic> toJson() => _$NovostToJson(this);
}
